1-introduction/introduction_Task5.py

```diff
@@ -0,0 +1,6 @@
+def get_sum(seq, n):
```

Comment:
Cosmetis: The name of the function does not reflect what it does in my opinion.

Reply:
Fixed